docs/hypernode-platform/mysql/how-to-use-mysql-on-hypernode.md

```diff
@@ -49,23 +49,23 @@ Port 3306 is fire-walled on all Hypernodes to prevent hackers and bruteforces fr
 
 First check which IP addresses have been whitelisted already, if any.
 
-```bash
+```console
```

Comment:
`console` should only be used when there's a prompt in the example, like:

```console
$ echo "hello"
```

or 

```console
app@abcdef-example-magweb-cmbl:~$ echo "hello"
```